scipy/stats/_stats_py.py

```diff
@@ -4461,7 +4473,7 @@ def binary_search(n, n1, n2, side):
         pexact = hypergeom.pmf(c[0, 0], n1 + n2, n1, n)
         pmode = hypergeom.pmf(mode, n1 + n2, n1, n)
 
-        epsilon = 1 - 1e-4
+        epsilon = 1 - 1e-14
```

Comment:
Feel free to suggest a different value, and I'll make the change. When I thought of `1e-14`, I was thinking this was an absolute tolerance, but it's relative, and it might be too tight for very small probability masses. [`binomtest`](https://github.com/scipy/scipy/blob/b5d8bab88af61d61de09641243848df63380a67f/scipy/stats/_binomtest.py#L301) uses something like `1e-7` and [this license-compatible implementation](https://github.com/brentp/fishers_exact_test/blob/master/src/cfisher.pyx) uses `1e-6`, but they are not using the tolerance in exactly the same way.

Reply:
Since you said you will update this part later, fine with me like this.